workspaces/announcements/package.json

```diff
@@ -59,5 +63,6 @@
     "*.{json,md}": [
       "prettier --write"
     ]
-  }
+  },
+  "packageManager": "yarn@4.4.1"
```

Comment:
This can be dropped as well, we use the same version everywhere and currently don't use this property.